src/components/content/order/serviceActions/ServiceActionStatusAlert.tsx

```diff
@@ -0,0 +1,138 @@
+                    <OrderProcessingStatus
+                        operationType={OperationType.SubmitAction}
+                        serviceOrderStatus={getServiceActionStatusPollingQuery.data}
+                        serviceId={serviceId}
+                        selectedServiceHostingType={serviceHostType}
+                    />
+                );
+            } else if (getServiceActionStatusPollingQuery.isError) {
+                if (serviceHostType === serviceHostingType.SERVICE_VENDOR) {
+                    return 'Status polling failed. Please visit MyServices page to check the status of the request and contact service vendor for error details.';
+                } else {
+                    return 'Status polling failed. Please visit MyServices page to check the status of the request';
+                }
+            } else if (
+                getServiceActionStatusPollingQuery.isPending ||
+                !getServiceActionStatusPollingQuery.data.isOrderCompleted
+            ) {
+                return 'Submitting, Please wait...';
+            }
+        }
+    }, [
+        createServiceActionRequest.isPending,
+        createServiceActionRequest.isError,
+        createServiceActionRequest.isSuccess,
+        createServiceActionRequest.error,
+        getServiceActionStatusPollingQuery.isSuccess,
+        getServiceActionStatusPollingQuery.data,
+        getServiceActionStatusPollingQuery.isError,
+        getServiceActionStatusPollingQuery.isPending,
+        serviceId,
+        serviceHostType,
+    ]);
+
+    const alertType = useMemo(() => {
+        if (createServiceActionRequest.isPending) {
+            return 'success';
+        } else if (createServiceActionRequest.isError || getServiceActionStatusPollingQuery.isError) {
+            return 'error';
+        } else if (createServiceActionRequest.isSuccess) {
+            if (
+                getServiceActionStatusPollingQuery.isSuccess &&
+                getServiceActionStatusPollingQuery.data.taskStatus.toString() === taskStatus.FAILED.toString()
+            ) {
+                return 'error';
+            } else if (
+                getServiceActionStatusPollingQuery.isSuccess &&
+                getServiceActionStatusPollingQuery.data.taskStatus.toString() === taskStatus.SUCCESSFUL.toString()
+            ) {
+                return 'success';
+            } else if (
+                getServiceActionStatusPollingQuery.isPending ||
+                getServiceActionStatusPollingQuery.data.taskStatus.toString() === taskStatus.IN_PROGRESS.toString()
+            ) {
+                return 'success';
+            }
+        }
+        return 'success';
+    }, [createServiceActionRequest, getServiceActionStatusPollingQuery]);
+
+    function getOrderSubmissionFailedDisplay(errorType: string, reasons: string[]) {
+        return (
+            <div>
+                <span>{errorType.length > 0 ? errorType : 'Service action request failed.'}</span>
```

Comment:
is it possible to also show the 'action name' in bold in the message. 
for example - 'Service action **backup** request failed'